portal/cookbook-guide.md

```diff
@@ -6,84 +6,261 @@ They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
 background knowledge.
 
-The following is a step-by-step guide to getting your cookbook idea
-hosted on the [Project Pythia Cookbooks Gallery](https://cookbooks.projectpythia.org).
+The following is a step-by-step guide to creating a new Cookbook and getting it hosted on the [Project Pythia Cookbook Gallery](https://cookbooks.projectpythia.org).
 
 Before you begin, ask yourself if the content you are developing for a cookbook would be better suited as an addition to an existing cookbook. The best place to discuss cookbook ideas is the [Project Pythia category of the Pangeo Discourse](https://discourse.pangeo.io/c/education/project-pythia/60).
 
+```{Note}
+These instructions assume that your goal is to contribute a new Cookbook to the community-maintained collection housed on the [Pythia Cookbook Gallery](https://cookbooks.projectpythia.org).
+
+Using the Pythia Cookbook template to create reproducible documents housed elsewhere is definitely possible! But we don't focus on that use case in this guide.
+```
+
+---
+
 ## Data access
 
 Before developing your cookbook, you should consider how it will access the data you plan to use. In loose order of preference, we recommend the following:
 
 1. Rely on data that is already freely available and accessible with tools in the ecosystem. Point to Foundations or other cookbooks for tool how-to guides if needed. Examples include the [CMIP6 Cookbook](https://projectpythia.org/cmip6-cookbook/) and the [CESM LENS on AWS Cookbook](https://projectpythia.org/cesm-lens-aws-cookbook/)
 1. Focus on representative subsets of data that can be packaged alongside the cookbook in-repo. An example is the [Landsat ML Cookbook](https://projectpythia.org/landsat-ml-cookbook/README.html)
-1. Wait for the Pythia team to explore cloud storage support via NSF JetStream or adjacent efforts
+1. Discuss your larger data storage needs with the Pythia team. We are currently experimenting with cloud object storage for Cookbooks via NSF JetStream2.
 1. Provide the tools and/or clear documentation for accessing the data that you have stored somewhere else
 
+---
+
 ## Use the template
 
 1. If you don't already have a GitHub account, create one by following the [Getting Started with GitHub guide](https://foundations.projectpythia.org/foundations/getting-started-github.html)
-1. On the [Cookbook Template repository](https://github.com/ProjectPythia/cookbook-template), click "Use this template&rarr;Create a new repository"
+1. On the [Cookbook Template repository](https://github.com/ProjectPythia/cookbook-template), click "Use this template &rarr; Create a new repository"
 1. Give your repository a descriptive name followed by `-cookbook` (e.g., `hydrology-cookbook`, `hpc-cookbook`, `cesm-cookbook`) and a description
 1. Choose "Include all branches" and create the repository. Your browser will be directed to the newly created repository under your GitHub account
```

Comment:
Small nit, but the option to "include all branches" occurs above the repository name selection

Reply:
Resolved: https://projectpythia.org/_preview/435/cookbook-guide.html#use-the-template